layouts/partials/header.html

```diff
@@ -41,7 +45,11 @@
             <ul class="nav-hamburger-list visibility-hidden">
                 {{ range .Site.Menus.main }}
                 <li class="nav-item">
-                    <a href="{{ .URL | absURL }}" {{- if .Params.NewPage -}}target="_blank"{{- end -}}>
```

Comment:
same here, might be worth to try @nalysius :)

Reply:
I've found a way on one line, I'll create another PR with only this change.